### PR TITLE
Added 'bbp/nse/morph-tool' package

### PR DESCRIPTION
**Pull request by arsenius7**
* morphio[-python] -> 2.0.3

### CODE REVIEW

File: bbp/nse/morph-tool/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    propagatedBuildInputs = with pythonPackages; [
+        click
+        functools32
+        numpy
+    ] ++ [
+        morphio-python
+    ];
+
+    preConfigure = ''
+        sed -i "s/numpy>=1.14/numpy>=1.13/g" setup.py
```
Comment (arsenius7) [inline bbp/nse/morph-tool/default.nix]: Is it OK to downgrade required `numpy` version for `morph-tool`?
Can we do in `morph-tool` repo?

File: bbp/nse/morphio-python [hunk 2]
```diff
@@ -23,4 +23,12 @@ pythonPackages.buildPythonPackage rec {
...
       stdenv
     ];
- }
+
+    propogatedBuildInputs = [
+      pythonPackages.numpy
+    ];
+
+    preConfigure = ''
+        sed -i "s/'numpy>=1.14.1'//g" setup.py
```
Comment (arsenius7) [inline bbp/nse/morphio-python]: Same question as with `morph-tool`: is it OK to downgrade required `numpy` version?
Also, in this case I have not managed to make `morphio-python` recognize `numpy` during installation at all (but `propogatedBuildInputs` should at least ensure the dependency is brought by Nix when needed).

File: bbp/nse/morph-tool/default.nix (hunk 1 above)
Comment (wizmer) [reply bbp/nse/morph-tool/default.nix]: Can't we upgrade nix numpy instead ?

Comment (arsenius7) [reply bbp/nse/morph-tool/default.nix]: If you feel brave. ^^
As far as I recall, it's a part of "base" distribution with all implications of that.

Comment (wizmer) [reply bbp/nse/morph-tool/default.nix]: I feel brave today, I'll try.

**Pull request merged**